tests/system/action/meeting/test_clone.py

```diff
@@ -259,11 +261,82 @@ def test_clone_user_ids_and_admin_ids(self) -> None:
                 "username": "new_default_group_old_admin_user",
                 "group_$_ids": ["1", "2"],
                 "group_$1_ids": [2],
-                "group_$2_ids": [3, 4],
+                "group_$2_ids": [4, 3],
+                "meeting_ids": [1, 2],
+            },
+        )
+
+    def test_clone_new_committee_and_user_with_group(self) -> None:
+        self.set_models(self.test_models)
+        self.set_models(
+            {
+                "user/13": {
+                    "username": "user_from_new_committee",
+                    "group_$_ids": ["1"],
+                    "group_$1_ids": [1],
+                    "meeting_ids": [1],
+                },
+                "group/1": {"user_ids": [13]},
+                "committee/2": {"organization_id": 1},
+                "organization/1": {"committee_ids": [1, 2]},
+                "meeting/1": {"user_ids": [13]},
+            }
+        )
+        response = self.request(
+            "meeting.clone",
+            {
+                "meeting_id": 1,
+                "user_ids": [13],
+                "committee_id": 2,
+            },
+        )
+        self.assert_status_code(response, 200)
+        self.assert_model_exists("meeting/2", {"committee_id": 2, "user_ids": [13]})
+        self.assert_model_exists(
+            "committee/2", {"user_ids": [13], "organization_id": 1, "meeting_ids": [2]}
+        )
+        self.assert_model_exists(
+            "user/13",
+            {
+                "username": "user_from_new_committee",
+                "committee_ids": [2],
```

Comment:
You should assert, that the user belongs to group/2 of meeting/2 (I assume the cloned objects in both cases have the id 2) and group/2 having the user/13 as member.

Reply:
Done.

Reply:
In the meeting 2 it is group 3 is the default group. group 2 is the admin group in meeting 1.